synapse/crypto/keyring.py

```diff
@@ -815,87 +814,67 @@ async def _fetch_keys(
 
         async def get_key(key_to_fetch_item: _FetchKeyRequest) -> None:
             server_name = key_to_fetch_item.server_name
-            key_ids = key_to_fetch_item.key_ids
 
             try:
-                keys = await self.get_server_verify_key_v2_direct(server_name, key_ids)
+                keys = await self.get_server_verify_key_v2_direct(server_name)
                 results[server_name] = keys
             except KeyLookupError as e:
-                logger.warning(
-                    "Error looking up keys %s from %s: %s", key_ids, server_name, e
-                )
+                logger.warning("Error looking up keys from %s: %s", server_name, e)
             except Exception:
-                logger.exception("Error getting keys %s from %s", key_ids, server_name)
+                logger.exception("Error getting keys from %s", server_name)
 
         await yieldable_gather_results(get_key, keys_to_fetch)
         return results
 
     async def get_server_verify_key_v2_direct(
```

Comment:
Possibly this should be `get_server_verify_keyS_v2_direct` plural now?

Reply:
```suggestion
    async def get_server_verify_keys_v2_direct(
```

Reply:
it was just as capable of fetching multiple keys before, but sure